cfme/configure/access_control.py

```diff
@@ -1405,6 +1407,28 @@ def set_quota(self, **kwargs):
             self.obj_type, self.name))
         assert view.is_displayed
 
+    @property
+    def quota(self):
+        view = navigate_to(self, 'Details')
+        quotas = {
+            'cpu': 'Allocated Virtual CPUs',
+            'memory': 'Allocated Memory in GB',
+            'storage': 'Allocated Storage in GB',
+            'num_vms': 'Allocated Number of Virtual Machines',
+            'templates': 'Allocated Number of Templates'
+        }
+
+        for field in quotas.keys():
```

Comment:
OPTIONAL: when you iterate over dict, python uses keys() be default.
So,
for key in some_dict.keys()  equal to for key in some_dict